system/cmake/sdl2-config.cmake

```diff
@@ -0,0 +1,16 @@
+set(SDL2_INCLUDE_DIRS "${EMSCRIPTEN_SYSROOT}/include/SDL2")
```

Comment:
I wonder if we can use the upstream `sdl2-config.cmake` file here rather than creating a new one.   I do like that this will add the needed link flags though.. its kind of a neat idea.